### DISCUSSION TITLE
mistake?

### DISCUSSION THREAD

**Discussion by MariaAssiduo** [gwastro/pycbc#3888] [Q&A]
https://github.com/gwastro/pycbc/blob/c0e2064ab7ec16e9819e85e537a0404969dfb713/pycbc/filter/matchedfilter.py#L666

Can I assume here that the comment is:

""Compute the match maximized over polarization phase.

 In contrast to compute_max_snr_over_sky_loc_stat this function performs no maximization over orbital phase [...] 

Otherwise It's not clear to me in contrast to which statistic?

Thanks.

**Comment by spxiwh** [comment]
@MariaAssiduo Thanks for raising this. I opened a PR here to hopefully make this clearer https://github.com/gwastro/pycbc/pull/3891. Please comment there if it's not! ... And feel free to raise more discussions/issues wherever documentation is unclear! We want the documentation to be useful, but rely on feedback to make that happen.
